### DISCUSSION TITLE
API rule - list records in relation

### DISCUSSION THREAD

**Discussion by bmingles** [pocketbase/pocketbase#3856] [Q&A] [answered]
I'm trying to configure api rules for a `tenants` collection based on whether the logged in user belongs to a tenant.

* My `users` collection has a `tenants` relationship field configured for `Multiple` tenant records
* I have a `tenants` collection with a list / search api rule of `@request.auth.tenants.id ?= id`

When I login via the REST api, the record returned contains a `tenants` array showing the ids I'd expect, but when I request `tenants` with this token, I get an empty result set. 

I have enabled the `--debug` flag and see the following query:

```
SELECT `tenants`.* FROM `tenants` WHERE (('' = [[tenants.id]] OR [[tenants.id]] IS NULL)) LIMIT 30
```

It looks like the `@request.auth.tenants.id` part of my rule is yielding an `''` even though my user has 2 tenantids associated with it.

**Comment by ganigeorgiev** [comment, marked as answer]
Double check that you are sending the request with valid `Authorization: AUTH_TOKEN` header and that the user associated with the token has non-empty `tenants`?

If that's not the case, please provide a code sample showing how you are sending the request or at least some steps to reproduce (_you can also export your collection configurations as json from Admin UI > Settings > Export collections_).

**Reply by bmingles** [reply]
It does looks like I had mistyped something in my script where I wasn't passing a valid token. I received a 200 response, so I assumed my auth had worked.
